content/spin/kv-store.md

```diff
@@ -23,7 +23,7 @@ enable_shortcodes = true
 
 ## Key Value Storage With Spin Applications
 
-Spin applications are best suited for event-driven, stateless workloads that have low-latency requirements. Keeping track of the application's state (storing information) is an integral part of any useful product or service. For example, users (and the business) will expect to store and load data/information at all times; during an application’s execution. In [Spin v0.9.0](https://www.fermyon.com/blog/spin-v09), we've added support for key/value storage. This is a storage option for applications that need data in the form of key/value pairs and are satisfied by a BASE consistency model. Workload examples include general value caching, session caching, counters, and serialized application state.
+Spin applications are best suited for event-driven, stateless workloads that have low-latency requirements. Keeping track of the application's state (storing information) is an integral part of any useful product or service. For example, users (and the business) will expect to store and load data/information at all times; during an application’s execution. From [Spin v0.9.0](https://www.fermyon.com/blog/spin-v09), onwards we've added support for key/value storage. This is a storage option for applications that need data in the form of key/value pairs and are satisfied by a BASE consistency model. Workload examples include general value caching, session caching, counters, and serialized application state.
```

Comment:
At this point maybe remove the version qualification?  We are not really making an effort to keep the docs relevant to 0.8 and earlier, so maybe treat K/V as just a feature that we have.

Reply:
Done